edit_book.php

```diff
@@ -0,0 +1,44 @@
+<?php
+include 'database.php';
+
+if (isset($_GET['id'])) {
+    $id = $_GET['id'];
```

Comment:
Check the result of mysqli_query before using mysqli_fetch_assoc to avoid warnings or errors.